src/chat/backend.rs

```diff
@@ -154,8 +160,8 @@ impl<S> http::Codec<S> for AuthCodec {
             if let Some(ref header) = self.destination.override_host_header {
                 e.add_header("Host", header).unwrap();
             }
-            ok(write_json_request(e,
-                &Auth(&serialize_cid(&self.conn_id), &i)))
+            let cid = PubCid(self.conn_id.clone(), self.runtime_id.clone());
```

Comment:
Maybe it would be better to make `runtime_id`(`server_id`) static variable
as it generated once and used in many places in chat?..